third_party/lua/loslib.c

```diff
@@ -135,6 +127,7 @@ __static_yoink("lua_notice");
 
 #define LUA_TMPNAMBUFSIZE	128
 
+// [jart]
```

Comment:
i think we can probably take the comment out since it's mentioned in README.cosmo - upstream doesn't need this because they just hardcode `"/tmp"`.

Reply:
Please see the big comment below.